techniques/css/C42.html

```diff
@@ -135,14 +133,14 @@ <h4>The CSS</h4>
 			
 			</section>
 			<section class="example" id="example-2-using-a-display-value-of-flex-and-min-height-/-width">
-				<h3>Example 2: Using a <code class="language-css">display</code> value of <code class="language-css">flex</code> and <code class="language-css">min-height</code> / <code class="language-css">width</code></h3>
-				<p>The second example uses <code class="language-css">min-width</code> and <code class="language-css">min-height</code> on the targets (the linked numbers in the pagination menu) and not on the parent container, thereby meeting this target spacing success criterion and incidentally also the AAA Success Criterion 2.5.5 Target Size.</p>
+				<h3>事例 2: <code class="language-css">display</code> 値に <code class="language-css">flex</code> 及び <code class="language-css">min-height</code> / <code class="language-css">width</code> を使用する</h3>
```

Comment:
この訳だと `display: flex min-height width` というように読めますが、 `display: flex` と `min-height/width:` を使うという意味ですね。ちなみに min-height / width というのは min-height and min-width という意味のはずです (width プロパティも存在するんだからちゃんと書けばいいのに……)。

and って「かつ」としても良いのでしたっけ? よければこうかなと:

```suggestion
				<h3>事例 2: <code class="language-css">display</code> の値に <code class="language-css">flex</code> を使用し、かつ <code class="language-css">min-height</code> / <code class="language-css">width</code> を使用する</h3>
```

Reply:
> and って「かつ」としても良いのでしたっけ?

特に問題はないです…ということで、コミットします。